services/app-web/src/components/SubmissionSummarySection/ContractDetailsSummarySection/ContractDetailsSummarySection.tsx

```diff
@@ -309,7 +309,7 @@ export const ContractDetailsSummarySection = ({
                         ? new Date(
                               documentDateLookupTable.previousSubmissionDate
                           )
-                        : undefined
```

Comment:
Curious why we had to switch from undefined to nulls here. Totally fine but I'm curious. in general I only expect to see nulls when dealing directly with Apollo GQL data

Reply:
Made this choice for React reasons, not GQL. Because this was previously an optional prop, the component is used in about 6 places, and the bug was that the date was present in some expected places and not in others it was very for me difficult to tell if data was coming back unexpected as undefined OR if it was just a case where the prop wasn't passed in at all intentionally.

I changed to a required prop. That means places using this component have to be clear when there is no previousSubmissionDate using explicit null